### ISSUE TITLE
Kuwait's Dawn/Dusk skins use old icons

### ISSUE THREAD

**Issue by MoltonMontro** [SmartlyDressedGames/Unturned-3.x-Community#3679]
The **Dawn Zubeknakov**, **Dusk Zubeknakov**, **Dawn Maplestrike**, and **Dusk Maplestrike** (4 skins total) all use outdated inventory icons. They show the iron sights as non-skinned default colors, rather than the custom fallback attachment texture they actually have in-game.

2 examples pictured below. Here's what it is currently shown. Notice the iron sights: 
![Dawn Maplestrike, current.](https://user-images.githubusercontent.com/7608445/215863688-815f5084-0f7c-46d2-b927-fd760d28eb21.png) ![Dusk Zubeknakov, current.](https://user-images.githubusercontent.com/7608445/215863721-2bde3fc8-b5e0-4776-965c-0e9ea589e328.png)

Here's what it should actually be, and what's shown correctly if the icon has been manually generated with the F1 menu tools:
![Dawn Maplestrike, new.](https://user-images.githubusercontent.com/7608445/215863497-e53c2f82-7f81-4a37-8301-fdc4d66d566d.png) ![Dusk Zubeknakov, new.](https://user-images.githubusercontent.com/7608445/215863504-15762f3b-fc1c-4e82-b869-56ae3b8817b6.png)

_**Note:** "outdated" might not be the most accurate term. I'm just assuming they had been generated before a fallback attachment texture had been provided. It's possible this was caused by a different issue entirely._

**Comment by SDGNelson** [comment]
Good point that there are some discrepancies. Unfortunately I think there are a variety of skins that got minor adjustments without new icons. I think what should happen here rather than a individual fix is reworking how the skin icons are captured to automatically keep them all in sync, similar to the new WIP-ish cosmetic previews. (cosmetic previews intended to replace icons)

**Comment by MoltonMontro** [comment]
> similar to the new WIP-ish cosmetic previews. (cosmetic previews intended to replace icons)

A similar system should probably be added for cosmetics/skins _with mythical effects_ too. E.g., aren't the icons for **Auras** still manually created, similar to the old outfit previews?

**Comment by SDGNelson** [comment]
Unfortunately the auras are complicated. Hard to automatically measure a good bounding size, and the capture doesn't handle transparency well
